src/atsu/src/components/InputValidationFansA.ts

```diff
@@ -0,0 +1,70 @@
+//  Copyright (c) 2022 FlyByWire Simulations
+//  SPDX-License-Identifier: GPL-3.0
+
+import { AtsuStatusCodes } from '../AtsuStatusCodes';
```

Comment:
Regex can do the hard work on all these too.

Reply:
Done